source/scss/brannan.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ * Brannan
+ *
+ */
+@import 'shared';
+
+// mixin to extend lark filter
+// @mixin brannan
+// @param $filters... {filter} - Zero to many css filters to be added
+// @example
+//   img {
+//     @include brannan;
+//   }
+//   or
+//   img {
+//     @include brannan(blur(2px));
+//   }
+//   or
+//   img {
+//     @include brannan(blur(2px)) {
+//       /*...*/
+//     };
+//   }
+@mixin brannan($filters...) {
+  @extend %filter-base;
+  filter: sepia(.2) brightness(1.1) contrast(1.3) saturate(.8) $filters;
+
+  &::after {
+    background: rgba(46, 0, 67, 0.61);
```

Comment:
if you make the changes above, try a more magenta color such as `rgba(66, 10, 54, 0.65);` (again, you did an awesome job and I have the easy task of testing and tweaking)